mutiny-core/src/nodemanager.rs

```diff
@@ -1052,14 +1055,115 @@ impl<S: MutinyStorage> NodeManager<S> {
             return Err(MutinyError::WalletOperationFailed);
         };
 
+        // If we are in safe mode, we don't create payjoin sessions
+        let pj = if self.safe_mode {
+            None
+        } else {
+            let context = payjoin::receive::EnrollContext::from_relay_config(
+                "http://localhost:8080",
+                "AQAg7YjKSn1zBziW3LvPCQ8X18hH0dU67G-vOcMHu0-m81AABAABAAM",
+                "http://localhost:8080", // Ignore and pass directly to relay for now
+                                         //Some("c53989e590b0f02edeec42a9c43fd1e4e960aec243bb1e6064324bd2c08ec498")
+            );
+            let pj_uri = format!("{}{}", "http://localhost:8080/", context.subdirectory());
+            log_debug!(self.logger, "{pj_uri}");
+            let wallet = self.wallet.clone();
+            // run await payjoin task in the background as it'll keep polling the relay
+            wasm_bindgen_futures::spawn_local(async move {
+                let wallet = wallet.clone();
```

Comment:
We'll probably want some way to stop this. A stop channel should be available to use but might need to also stop it if one of the payment methods in the bip21 completes. Though we haven't linked them together yet so that might be hard to do currently.

Reply:
https://github.com/MutinyWallet/mutiny-node/blob/e11cdc341e935b8dd1ae7a59f3449f2577184004/mutiny-core/src/nodemanager.rs#L969-L975

Reply:
We should also be able to start this from persistent storage if the wallet goes offline and wants to check once it's back open online